### ISSUE TITLE
For Improvement

### ISSUE THREAD

**Issue by zhamri** [STIW2024/Fruits-Online-Shop-FOS-#12]
Please watch the [Presentation Video](https://github.com/STIW2024/Fruits-Online-Shop-FOS-/wiki/Youtube-link-Presentation) and test the [Web Application](https://github.com/STIW2024/Fruits-Online-Shop-FOS-/wiki/Website). Then give some good comments/ideas for the betterment of this project.

**Comment by JohnWick27** [comment]
Yeah, Nice website with good functionality, keep it upgrading. it can be one of your main Project. Hopefully.

**Comment by ChoyCheeWei** [comment]
Good website. nice design and full functionality

**Comment by NgZhengYi** [comment]
Nice website, very easy to be use to purchase item

**Comment by jairam0905** [comment]
Very beautiful design applied from your team. Well done and keep it up!

**Comment by ernie2796** [comment]
Great website with great functionality :+1: The hover button design is so cool. It's my first time seeing such design. Yet, it would be nice if the button size is a bit bigger. Overall, it's a good website! :+1:

**Comment by kopilim96** [comment]
Good Website designed

**Comment by afiqznw** [comment]
Great website and easy to use...keep up the good work...

**Comment by Koksheng** [comment]
Amazing website!
